src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,150 @@
+package frc.robot.subsystems.climber;
+
+import static frc.robot.subsystems.climber.ClimberConstants.*;
+
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.team3015.subsystem.FaultReporter;
+import frc.lib.team6328.util.TunableNumber;
+import frc.robot.subsystems.subsystem.SubsystemIOInputsAutoLogged;
+
+import org.littletonrobotics.junction.Logger;
+
+/**
+ * Models a generic subsystem for a rotational mechanism. The other subsystems defined in this
+ * library aren't good examples for typical robot subsystems. This class can serve as an example or
+ * be used for quick prototyping.
+ */
+public class Climber extends SubsystemBase {
+
+  // these Tunables are convenient when testing as they provide direct control of the subsystem's
+  // motor
+  private final TunableNumber leftMotorPower = new TunableNumber("Climber/leftPower", 0.0);
+  private final TunableNumber leftMotorCurrent = new TunableNumber("Climber/leftCurrent", 0.0);
+  private final TunableNumber leftMotorPosition = new TunableNumber("Climber/leftPosition", 0.0);
+
+  private final TunableNumber rightMotorPower = new TunableNumber("Climber/rightPower", 0.0);
+  private final TunableNumber rightMotorCurrent = new TunableNumber("Climber/rightCurrent", 0.0);
+  private final TunableNumber rightMotorPosition = new TunableNumber("Climber/rightPosition", 0.0);
+
+  private final SubsystemIOInputsAutoLogged inputs = new SubsystemIOInputsAutoLogged();
+  private ClimberIO io;
+
+  /**
+   * Create a new subsystem with its associated hardware interface object.
+   *
+   * @param io the hardware interface object for this subsystem
+   */
+  public Climber(ClimberIO io) {
+
+    this.io = io;
+
+    // Create a Shuffleboard tab for this subsystem if testing is enabled. Add additional indicators
+    // and controls as needed.
+    if (TESTING) {
+      ShuffleboardTab tab = Shuffleboard.getTab(CLIMBER);
+      tab.add(CLIMBER, this);
+    }
+
+    FaultReporter.getInstance().registerSystemCheck(CLIMBER, getSystemCheckCommand());
+  }
+
+  // FIXME: Update inputs needs to be fixed, currently have no idea what to do
+  /**
+   * The subsystem's periodic method needs to update and process the inputs from the hardware
+   * interface object.
+   */
+  @Override
+  public void periodic() {
+    io.updateInputs(inputs);
+    Logger.processInputs("Climber", inputs);
+
+    // when testing, set the motor power, current, or position based on the Tunables (if non-zero)
+    if (TESTING) {
```

Comment:
Again, with Tuner X, I'm not sure if this approach is needed anymore. (It was definitely how we did things last year.)